src/IB/ConstraintIBMethod.cpp

```diff
@@ -169,6 +169,9 @@ ConstraintIBMethod::ConstraintIBMethod(const std::string& object_name,
       d_FuRMoRP_current_time(0.0),
       d_FuRMoRP_new_time(0.0),
       d_vol_element(d_no_structures, 0.0),
+      d_structure_vol(d_no_structures, 0.0),
+      d_structure_mom(d_no_structures, std::vector<double>(3, 0.0)),
```

Comment:
Well now that you mention it...

Basically any vector that is supposed to be a 3-space vector could be Vector3d, since that is a fixed-sized array.